products/windows.md

```diff
@@ -9,236 +9,259 @@ releasePolicyLink: https://learn.microsoft.com/lifecycle/products/?terms=Windows
 activeSupportColumn: true
 releaseColumn: true
 releaseDateColumn: true
-releaseLabel: "Windows __RELEASE_CYCLE__"
 
 identifiers:
 -   cpe: cpe:2.3:o:microsoft:windows
 -   cpe: cpe:/o:microsoft:windows
 
 releases:
--   releaseCycle: "10, version 22H2 (E)"
+-   releaseCycle: "10-22H2"
```

Comment:
By merging the two releases isn't there a risk that future changes makes us split it again, which will imply a breaking change of the API (similar to #3791) ?

Reply:
Same comment for (E)(W) versions below.